### PR TITLE
Added utility file for testing

### PR DESCRIPTION
**Pull request by kiryx**
This is a temporary solution for our problems with reusable unit testing functions

### CODE REVIEW

File: tests/test.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ *   Free Software Foundation, Inc.,                                         *
+ *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.               *
+ *****************************************************************************/
+
+#ifndef PAGMO_TEST_H
+#define PAGMO_TEST_H
+
+#include "../src/pagmo.h"
+
+#define PRINT_VEC(x) std::cout<<#x "[] = "; for(unsigned int __i=0;__i<(x).size();__i++) std::cout<<(x)[__i]<<" "; std::cout<<std::endl;
```
Comment (neduard) [inline tests/test.h]: Shouldn't this be enclosed in a do{} while(); block?

File: tests/test.h [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+bool is_eq(double d1, double d2, double eps=10e-9)
+{
+	return (fabs(d1 - d2) <= eps);
+}
+
+bool is_eq_vector(const pagmo::fitness_vector &f1, const pagmo::fitness_vector &f2, double eps=10e-9)
+{
+	if(f1.size() != f2.size()) return false;
+		
+		for(pagmo::fitness_vector::size_type i = 0; i < f1.size(); i++) {
```
Comment (neduard) [inline tests/test.h]: nitpick: remove extra indentation

File: tests/test.h (hunk 1 above)
Comment (kiryx) [inline tests/test.h]: Why ?

Comment (kiryx) [inline tests/test.h]: You're right

File: tests/test_shifted.cpp [hunk 3]
```diff
@@ -55,7 +46,7 @@ int shifted_test1(){
 	fitness_vector f_shifted = prob_shifted.objfun(p1_shifted_space);
 	constraint_vector c_expected = prob.compute_constraints(p1_original_space);
 	constraint_vector c_shifted = prob_shifted.compute_constraints(p1_shifted_space);
-	if(is_eq(f_shifted, f_expected, EPS)){
+	if(is_eq_vector(f_shifted, f_expected, EPS)){
```
Comment (neduard) [inline tests/test_shifted.cpp]: note: EPS 10e-9 is default argument for is_eq_vector; so we can just use `is_eq_vector(f_shifted, f_expected)` right?

**Pull request merged**